src/ranch_ssl.erl

```diff
@@ -116,6 +116,7 @@ listen(TransOpts) ->
 	SocketOpts = maps:get(socket_opts, TransOpts, []),
 	case lists:keymember(cert, 1, SocketOpts)
 			orelse lists:keymember(certfile, 1, SocketOpts)
+			orelse lists:keymember(user_lookup_fun, 1, SocketOpts)
```

Comment:
Let's keep the alphabetical order here since I can only imagine that the number of such options will keep increasing.